src/framework/uicomponents/qml/MuseScore/UiComponents/StyledComboBox.qml

```diff
@@ -127,29 +170,29 @@ ComboBox {
                     when: index === 0
 
                     PropertyChanges { target: delegateBackgroundRect; topLeftRadius: 4
-                                                                      topRightRadius: 4
-                                                                      bottomLeftRadius: 0
-                                                                      bottomRightRadius: 0 }
+                        topRightRadius: 4
+                        bottomLeftRadius: 0
+                        bottomRightRadius: 0 }
```

Comment:
auto indentation (Ctrl+A Ctrl+I)

Reply:
Is there a reason for having `PropertyChanges { target: delegateBackgroundRect; topLeftRadius: 4` all on the same line, or could it be:

```qml
PropertyChanges {
    target: delegateBackgroundRect;
    topLeftRadius: 4
```

Also, could the closing `}` go on a new line after `bottomRightRadius: 0`?

Reply:
reformatted the code